src/ArborX_Config.hpp.in

```diff
@@ -12,6 +12,9 @@
 #ifndef ARBORX_CONFIG_HPP
 #define ARBORX_CONFIG_HPP
 
+#cmakedefine ArborX_VERSION_STRING "@ArborX_VERSION_STRING@"
+#cmakedefine ArborX_GIT_COMMIT_HASH "@ArborX_GIT_COMMIT_HASH@"
```

Comment:
All caps please (c.f. https://github.com/arborx/ArborX/pull/42#discussion_r281214386 :D)

Reply:
So, the funny thing, I tried all caps first. I had
```CMake
set(ArborX_VERSION_STRING "0.9 (dev)")
```
in `CMakeLists.txt`, and
```c++
#cmakedefine ARBORX_VERSION_STRING "@ArborX_VERSION_STRING@"
```
here. But... it did not define it in the resulting `ArborX_Config.hpp` (`#undef`). I have no clue why that is.

Reply:
[`CMake`](https://cmake.org/cmake/help/v3.14/command/configure_file.html?highlight=cmakedefine) says:
> ```
> #cmakedefine VAR ...
> ```
> will be replaced with either:
> ```
> #define VAR ...
> ```
> or:
> ```
> /* #undef VAR */
> ```
> depending on whether `VAR` is set in `CMake` to any value not considered a false constant by the `if()` command.

Reply:
Thanks @masterleinad! I was too lazy to look it up, but it confirmed what I suspected: you can't have your cake and eat it too. I.e., I would have had to declare the variable as `ARBORX_VERSION_STRING` in CMake too, which I did not want to do :|

Reply:
I tried to avoid that in #42, too. But then someone complained. :wink:

Reply:
Oh, yeah, I did not realize that was the exactly same case 🤦‍♂ . I apologize. I'll change it here too.